lib/csv_importer/base.rb

```diff
@@ -32,6 +32,8 @@ def run
 
         target_data_model.insert_all(record_hashes) unless record_hashes.empty?
       end
+
+      Rake::Task["dfe:analytics:import_entity"].invoke(target_data_model.table_name) if DfE::Analytics.enabled?
```

Comment:
Would be tempted to wrap calling this rake task with our own model (`DfEAnalytics.sync(SomeClass)`) - we could then try stubbing the call to that class in the tests and potentially that would work on CI 🤷